src/utils/Client.util.ts

```diff
@@ -292,7 +292,7 @@ export function getClientOptions(
     isKeepAlive: clientOptions?.isKeepAlive,
     logger: clientOptions?.logger ?? defaultLoggerOptions,
     actor: clientOptions?.actor,
-    daprApiToken: clientOptions?.daprApiToken,
+    daprApiToken: clientOptions?.daprApiToken ?? process.env.DAPR_API_TOKEN,
```

Comment:
We can move this to [Settings.util.ts](https://github.com/dapr/js-sdk/blob/0fb83c1fda6a47e3f9a5753d24e6b0dcfe311985/src/utils/Settings.util.ts#L93) for consistency.

Reply:
I thought about the same thing, but it seemed like we only use it when there is a default value. Let me try it out and see how it looks.

Reply:
Yeah, I noticed that in some cases we even have `""` as the default, which essentially is the same as not having one.